cmd/krel/templates/latest/rpm/cri-tools/cri-tools.spec

```diff
@@ -1,3 +1,4 @@
+%global debug_package %{nil}
```

Comment:
Reference: https://docs.fedoraproject.org/en-US/packaging-guidelines/Debuginfo/#_useless_or_incomplete_debuginfo_packages_due_to_other_reasons

Reply:
Yep, debug packages often are empty and such, nice :+1:

Reply:
Why did this comment show up twice? Weird :thinking: